appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -182,7 +186,7 @@ public void updateMenuState(int numSelectedProjects, int numProjects) {
         : MESSAGES.exportProjectMenuItem();
     fileDropDown.setItemHtmlById(WIDGET_NAME_EXPORTPROJECT, exportProjectLabel);
     fileDropDown.setItemEnabledById(WIDGET_NAME_EXPORTPROJECT, allowExport);
-    fileDropDown.setItemEnabled(MESSAGES.exportAllProjectsMenuItem(), allowExportAll);
+    fileDropDown.setItemEnabledById(MESSAGES.exportAllProjectsMenuItem(), allowExportAll);
```

Comment:
setItemEnabled works on the text of the item, whereas setItemEnabledById works on the name of the item. It's not clear to me why this was changed, and it may break in non-English locales.